source/MaterialXRender/ViewHandler.cpp

```diff
@@ -16,54 +16,30 @@ float ViewHandler::degreesToRadians(float degrees) const
     return (degrees * PI_VALUE / 180.0f);
 }
 
-void ViewHandler::setPerspectiveProjectionMatrix(float fov,
-                                                 float aspectRatio,
-                                                 float nearClipPlane,
-                                                 float farClipPlane)
+Matrix44 ViewHandler::createViewMatrix(const Vector3& eye,
```

Comment:
New view + projection computation utilities to match viewer. Will clean up viewer in a bit along with some other changes.